scripts/st2_bootstrap_selector.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+BASE_PATH="https://raw.githubusercontent.com/StackStorm/st2-packages/master/scripts/st2bootstrap"
+BOOTSTRAP_FILE='st2bootstrap.sh'
+
+DEBTEST=`lsb_release -a 2> /dev/null | grep Distributor | awk '{print $3}'`
```

Comment:
this is incorrect: `lsb_release` can be present on rpm distro. So I find this check not sufficient...
Why not to try `[ -f /etc/debian_version]` it seems less prone to errors.